ui/test/nuxeo-tooltip.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html, fixture, flush } from '@nuxeo/testing-helpers';
+import '../widgets/nuxeo-tooltip.js';
+
+suite('nuxeo-tooltip', async () => {
+  test('Should add a _tooltip property', async () => {
+    const tooltip = await fixture(
+      html`
+        <nuxeo-tooltip>Hello</nuxeo-tooltip>
+      `,
+    );
+    await flush();
+
+    tooltip.show();
+    expect(tooltip._tooltip).to.not.be.null;
```

Comment:
Testing the internals (`_tooltip != null`) is very valid!
But since we are handling visual stuff, it would also be interesting to check that we render what we expect 🙂

Reply:
I agree. Since `_tooltip` is a `paper-tooltip` itselft, we should also check that it is displayed (i.e. that calling `this._tooltip.show();` actually resulted in something).

Reply:
@Gabez0r exactly! working on the async challenges related, but that's the goal :)